src/Features/Core/Portable/QuickInfo/QuickInfoService.cs

```diff
@@ -28,6 +35,13 @@ public abstract class QuickInfoService : ILanguageService
             Document document,
             int position,
             CancellationToken cancellationToken = default)
+            => GetQuickInfoAsync(document, position, SymbolDescriptionOptions.Default, cancellationToken);
```

Comment:
1. should this have the same assert?
2. now that this is internal, can we make this non-virtual, and then fixup all derived types we own to override the new virtual?

Reply:
I'm not sure whether making a method non-virtual is a binary breaking change.

Reply:
A simple test shows it works. I'll make the change.